src/main/java/org/openstreetmap/atlas/geography/atlas/raw/slicing/changeset/SimpleChangeSet.java

```diff
@@ -0,0 +1,112 @@
+package org.openstreetmap.atlas.geography.atlas.raw.slicing.changeset;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+import org.openstreetmap.atlas.geography.atlas.raw.slicing.temporary.TemporaryEntity;
+import org.openstreetmap.atlas.geography.atlas.raw.slicing.temporary.TemporaryLine;
+import org.openstreetmap.atlas.geography.atlas.raw.slicing.temporary.TemporaryPoint;
+
+/**
+ * Records any additions, updates and deletions that occurred during point and way raw Atlas
+ * slicing. We try to keep this class as light-weight as possible. For additions, we rely on the
+ * {@link TemporaryEntity} objects to keep track of the bare minimum needed to create Atlas
+ * entities. For tag updates, we store a mapping between the identifier of the object and the added
+ * tags. For deletions, we store the set of identifiers to delete. We also keep a mapping between
+ * deleted and created lines to properly update any relations. No relation slicing is done with this
+ * class. We only update existing relations if a member line was deleted and split into two or more
+ * lines.
+ *
+ * @author mgostintsev
+ */
+public class SimpleChangeSet
```

Comment:
There are now 2 classes called SimpleChangeSet https://github.com/osmlab/atlas/blob/dev/src/main/java/org/openstreetmap/atlas/geography/atlas/changeset/SimpleChangeSet.java 
It doesn't seem like the one I have referenced is actually used, so might considering deleting that one.

Reply:
Good callout! There are a number of change sets in the Atlas codebase, some no longer applicable. I didn't want to use the one above since it had fields I would never use. In this case, I thought that creating a country-slicing specific change-set to make it specific and light-weight was the right way to go. Hopefully being in the ***/slicing package makes this more clear. Having said that, there are a number of classes that can and will be removed after this refactor is done. SimpleChangeSet above is a candidate. Thanks Mike!